torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19732,6 +19733,7 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
     ),
 ]
 op_db += opinfo.definitions.op_db
+op_db += hop_db
```

Comment:
@janeyx99

Reply:
This would imply that every test we run for ops in op_db, we _should_ run for hop_db. Is this true? If so, this would increase a lot of testing if it were not already the case.

@zou3519 would likely have an opinion on this.